components/instructions/programs/uxdProtocol.tsx

```diff
@@ -10,7 +10,149 @@ import { tryGetTokenMint } from '@utils/tokens';
 import { ANCHOR_DISCRIMINATOR_LAYOUT } from '@utils/helpers';
 
 export const UXD_PROGRAM_INSTRUCTIONS = {
+  EmXCGBmeZ7vTZu1NcuR5Cod8438aQdghhVa69zcBVF23: {
```

Comment:
What's this? It's some sort of serialised ix?

Reply:
the test mainnet program id